ws/ShardManager.ts

```diff
@@ -0,0 +1,34 @@
+import DiscordWebsocket, { ConnectionProperties } from './DiscordWebsocket';
+
+interface BotGateway {
+  session_start_limit: {
+    max_concurrency: number;
+    remaining: number;
+    reset_after: number;
+    total: number;
+  };
+  shards: number;
+  url: string;
+}
+
```

Comment:
This should really be part of structures

Reply:
I moved it, but if so, shouldn't the `ConnectionProperties` interface in DiscordWebsocket.ts also be defined in structures?

Reply:
ConnectionProperties can be left in DWS since it isn't exposed in REST